.luacheckrc

```diff
@@ -2,6 +2,13 @@ std = "ngx_lua"
 unused_args = false
 redefined = false
 max_line_length = 100
+
 exclude_files = {
     "apisix/cli/ngx_tpl.lua",
 }
+
+files["apisix/patch.lua"] = {
```

Comment:
We can solve in this way, which is consistent:
https://github.com/apache/apisix/blob/6e09d56e9c93666b52fccdeb6e1d3fb1178b678c/apisix/patch.lua#L310-L311